src/Analyzer.JsonRuleEngine/Schemas/ConfigurationsDefinition.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
Was the folder renamed intentionally? Just double checking

Reply:
Yes. When I created a class, it auto made the namespace be `.Schema` , but all the "Schemas" have namespace `.Schemas`

Reply:
I think the configurations stuff should be at Analyzer.Core project not in the Rule Engine since it's scoped to the entire tool not to a particular engine in the tool. Do we need to make interfaces for these types of things to have consistency across our engines?

Reply:
I don't know if we need an interface or not, but the config should probably be moved to Analyzer.Types.